### PR TITLE
refactor: Replaced telerik components with oxyplot in Views, …

### PR DESCRIPTION
**Pull request by ramateur**
…except AnalyticReports

### CODE REVIEW

**Review by drusteeby — COMMENTED** [review]
LGTM. Just gotta remove app.config changes

File: App.config
[the remark this answers is not in the document]
Comment (ramateur) [reply App.config:1]: Tell me please how to do it. Add it to gitignore?

[the remark this answers is not in the document]
Comment (drusteeby) [reply App.config:1]: Copy and paste the original and push it as a new commit.

[the remark this answers is not in the document]
Comment (ramateur) [reply App.config:1]: Done

**Review by drusteeby — APPROVED** [review]

**Comment by drusteeby** [comment]
This should remain open until @silahian has a chance to merge it.

**Comment by silahian** [comment]
before I merge this, @ramateur can you please make sure to follow these instructions:
https://github.com/silahian/VisualHFT/issues/2#issuecomment-1646988971
And then chack that all is ok?

**Comment by ramateur** [comment]
Ok, Ithink it can take a couple of days. I added dummy data to database, but launch cound not analytics report with it, it still displays No data (I changed date picker as you described) (
Maybe it is because I had to switch to Binance povider instead of BinanceUS. The former is not accessible for me. And switched pair to BTC-USDT, which is present both on Coinbase and Binance

**Comment by ramateur** [comment]
There is a bug in demoTradingCore Program.cs when chagnging BinanceUs provider to Binance, symbol pair to "BTC-USDT" and commenting string 31 //await InitializeCoinbase(). Occurs mismatch between key-value pair in dictionary. I worked it around replacing book.MarketSymbol with _SYMBOLS.First() value as key, but may be you have another point of view on it.

**Pull request merged**